cmake/compiler/icx/generic.cmake

```diff
@@ -5,3 +5,4 @@ if(DEFINED TOOLCHAIN_HOME)
 endif()
 
 find_program(CMAKE_C_COMPILER icx ${find_program_icx_args})
+find_program(CMAKE_GCOV gcov ${find_program_icx_args})
```

Comment:
Is "gcov" the correct utility with the Intel compiler?  This [link](https://software.intel.com/content/www/us/en/develop/documentation/cpp-compiler-developer-guide-and-reference/top/optimization-and-programming-guide/tools/pgo-tools/code-coverage-tool.html) indicates it might be "codecov".